### ISSUE TITLE
Custom thousands/decimal separator doesn't work

### ISSUE THREAD

**Issue by rimace** [koalyptus/TableFilter#494]
When setting the `thousands_separator` to '.' and `decimal_separator` to ',', it seems that it doesn't change its behavior.
This also appears when trying to set this configuration per column via `{ type: 'formatted-number', decimal: ',', thousands: '.' }`

**Comment by koalyptus** [comment]
What do you mean exactly? Do your values match the supplied definition? refer to this demo:
http://koalyptus.github.io/TableFilter/data-types.html

**Comment by rimace** [comment]
You're right! This example works (which nearly made me crazy :D ). I finally managed to find out what is really was. In the example (in the € column) there are numbers like "2.776,09" or "2.743,41" and filtering for ">1000" filters correctly there. But if the column contains numbers like "2.776" and "2.743" (so without a comma) it doesn't work anymore.
I tested it with the example and it's true, when removing the comma the filter thinks "." is the decimal separator even if it's configured as a thousands separator.
